### PR TITLE
[drc_task_common] Add RobotHeadUI to specify joint angles of head directly

### PR DESCRIPTION
**Pull request by garaemon**
Not yet integrated in teleop system.

### CODE REVIEW

**Comment by garaemon** [comment]
Updated to work on real robot

File: jsk_2015_06_hrp_drc/drc_task_common/euslisp/fc-executive.l [hunk 1]
```diff
@@ -524,6 +528,32 @@
...
+      (send self :look-at-angle-common)))
+  (:look-at-yaw-command-callback (msg)
+    (when (send smachine :state? :initial)
+      (send *robot* :angle-vector (send *ri* :state :reference-vector))
+      (send *robot* :head :neck-y :joint-angle (elt (send msg ::float_params) 0))
+      (send self :look-at-angle-common)))
+  (:look-at-angle-common ()
+    "Common method for :look-at-pitch-command-callback and :look-at-yaw-command-callback.
+It sends :angle-vector to robot and return 'finished' message to ocs"
+    (cond
```
Comment (garaemon) [inline jsk_2015_06_hrp_drc/drc_task_common/euslisp/fc-executive.l]: @mmurooka Is it correct use of head overwrite mode?

Comment (mmurooka) [inline jsk_2015_06_hrp_drc/drc_task_common/euslisp/fc-executive.l]: Yes.

**Comment by garaemon** [comment]
travis passed

**Pull request merged**